examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>It can be retrieved through the Braket console or programmatically through the device properties:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>